test/features/app/app.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const preq   = require('preq');
-const assert = require('../../utils/assert.js');
+const { assert } = require('../../utils/assert.js');
```

Comment:
I just figured since all services are forked off of this that it wouldnt break anything if it was changed but probably best to not even change it huh.

Reply:
The update path for services is to rebase the fork over the newer master of template, which will break things.

Reply:
OH yikes i didnt know that. good to know